protocol/x/subaccounts/keeper/subaccount.go

```diff
@@ -362,7 +358,8 @@ func (k Keeper) CanUpdateSubaccounts(
 		return false, nil, err
 	}
 
-	return k.internalCanUpdateSubaccounts(ctx, settledUpdates)
+	success, successPerUpdate = k.internalCanUpdateSubaccounts(ctx, settledUpdates)
+	return success, successPerUpdate, nil
```

Comment:
Planning to revert this update in a follow-up PR (currently function signatures are different so I had to introduce this change)

I avoided making all changes in one PR since it's too large and difficult to review (see [this PR](https://github.com/dydxprotocol/v4-chain/pull/677) if you want to see all changes I plan to make).